modules/creating-a-custom-ingress-controller.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// *ingress-controller-dnsmgt.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-custom-ingress-controller_{context}"]
+= Creating a custom Ingress Controller
+
+As a cluster administrator, you can create a custom Ingress Controller to manage DNS record lifecycle.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* DNS record created.
+
+.Procedure
+
+. Create an `IngressController` custom resource to set `dnsManagementPolicy`:
+
++
+[source,terminal]
+----
+oc -n openshift-ingress-operator patch ingresscontrollers/<name> --type=merge --patch='{"spec":{"endpointPublishingStrategy":{"loadBalancer":{"dnsManagementPolicy":"Unmanaged"}}}}'
+----
```

Comment:
This attempts to modify an existing Ingress Controller, we don't need it under this section.

Reply:
can you share the oc command i need to add , to create

Reply:
There is no direct way to create an Ingress Controller, you have to have a `yaml` for it that you `oc apply`. (which you have documented below)

Reply:
yes makes sense .

Reply:
I think instead we can say >> Create a CR file named sample.yaml that contains the following:

Reply:
nit: `sample-ingress.yaml`